src/styles/variables/responsive.scss

```diff
@@ -1,12 +1,8 @@
 $breakpoints: (
     phone: 480px,
-    tablet: 768px,
-    desktop: 1280px
+    tablet: 768px
 );
 
-// boxed view 1200px
-$boxed-width: 120rem;
```

Comment:
I think `boxed-width` should stay since it's being used throughout the site per Fred's designs.